modules/auxiliary/scanner/http/emby_version_ssrf.rb

```diff
@@ -0,0 +1,72 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
+
+  def initialize
+    super(
+      'Name' => 'Emby Version Checker',
+      'Description' => 'This module attempts to identify the version of an Emby Media Server running on a host. If you wish to see all the information available, set VERBOSE to true. Use in conjunction with emby_ssrf_scanner to locate devices vulnerable to CVE-2020-26948.',
+      'Author' => 'Btnz',
+      'License' => MSF_LICENSE,
+      'Disclosure Date' => 'September 1 2020',
+      'References' =>
+              [
+                ['CVE', '2020-26948'],
+                ['URL', 'https://github.com/btnz-k/emby_ssrf']
+              ]              
+    )
+
+    register_options(
+      [
+        Opt::RPORT(8096),
+        OptString.new('BASEPATH', [true, 'The base path, usually just /', '/']),
```

Comment:
Should be `TARGETURI` for consistency.
https://github.com/rapid7/metasploit-framework/blob/master/modules/exploits/example_webapp.rb#L78